includes/class-payment-gateways.php

```diff
@@ -115,10 +114,6 @@ public function _admin_warnings() {
             return;
         }
 
-        if ( ! wpbdp_get_option( 'payments-on' ) ) {
```

Comment:
If there aren't any plans with fees, we should nag about adding payment gateways. Let's keep this part.

Can you please go through and test every change on both a new and existing site?

Reply:
Tested on new and existing site. Removed notice that asked users to enable or disable payments.

- When a plan with a price is enabled and there are no active gateways, this is shown
![Screenshot 2021-11-10 164043](https://user-images.githubusercontent.com/121492/141123612-7b1f0f6d-9ac0-4663-83e3-48eba98279de.png)
- When a gateway is  enabled, normal checkout flow goes.
- Only enabled plans are shown on checkout

Reply:
Doing more tests on the checkout process

Reply:
It looks like this still return needs to be added back in along with the gateway check below it.